responses/de/HassGetState.yaml

```diff
@@ -1,4 +1,63 @@
 language: de
 responses:
   intents:
-    HassGetState: {}
+    HassGetState:
+      einzeln: |
+        {{ slots.name | capitalize }} ist {{ state.state_with_unit }}
+
+      einzeln_janein: |
+        {% if query.matched %}
+          Ja
+        {% else %}
+          Nein
+        {% endif %}
+
+      irgendeins: |
+        {% if query.matched %}
+          {% set match = query.matched | map(attribute="name") | sort | list %}
+          {% if match | length > 4 %}
+            Ja, {{ match[:3] | join(", ") }} and {{ (match | length - 3) }} weitere
```

Comment:
here "and" should be replaced with "und"

Reply:
nice catch :+1: